dask/dataframe/tests/test_groupby.py

```diff
@@ -3302,8 +3302,9 @@ def test_groupby_sort_true_split_out():
     M.sum(ddf.groupby("x", sort=False), split_out=2)
 
     # Warns for sort=None
-    with pytest.warns(FutureWarning, match="split_out>1"):
-        M.sum(ddf.groupby("x"), split_out=2)
+    with pytest.warns(None):
+        ddf.groupby("x").sum(split_out=2)
+        ddf.groupby("x").agg("sum", split_out=2)
```

Comment:
Shall we remove the context entirely?

Reply:
That's a pattern we use in pandas when we want to show that it doesn't warn anymore, so I would like to keep It, it's a bit more explicit

Reply:
Okay, no issue with that. Thanks!